Forms/MainForm.cs

```diff
@@ -1043,14 +1244,46 @@ private bool ValidateInputFile()
             return true;
         }
 
+        private bool ValidatePresetFile(String[] lines)
+        {
+            if (lines.Length > 0 && (lines[0].Equals("MMR Preset File [" + AssemblyVersion + "]") || lines[0].Equals("MMR Preset File [dev]")))
+            {
+                return true;
+            }
+            else
+            {
+                tbPreset.Text = "";
+
+                if (PresetInit)
+                {
+                    File.Delete(".\\" + initSettingsFilename + PresetExt);
```

Comment:
This means when a new version is run, it will delete the saved settings that are there if there are any. Unless they are somehow up to date with the current version. This is basically just error handling.

Reply:
But why is that a desirable effect? Users will have the settings they play with wiped with every new release? Maybe change it to every new 1.X release wipes the settings? That way new features will wipe them, but bug fix releases will not?